.github/workflows/workflow.yml

```diff
@@ -0,0 +1,30 @@
+name: SwiftUICoordinator
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+    env:
+      PROJECT: Example/SwiftUICoordinatorExample/SwiftUICoordinatorExample.xcodeproj
+      DEVELOPER_DIR: /Applications/Xcode.app/Contents/Developer
+      SCHEME: SwiftUICoordinatorExample
+      DESTINATION: platform=iOS Simulator,name=iPhone 14
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Select Xcode
+      run: sudo xcode-select --switch $DEVELOPER_DIR
+    - name: Brew update
+      run: brew update
```

Comment:
Also, brew update only updates the formulae definitions, does not upgrade them. And since you're not using Brew in the build phase, I think it's safe to remove it